### PR TITLE
Prepares watchmaker 0.10.2 release

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

**Comment by eemperor** [comment]
go codebuild go

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+**Released**: 2018.09.27
+
+**Summary**:
+
+*   Adds a gitlab-ci pages config to build Watchmaker docs
+*   Uses new hosting location to retrieve Salt packages
+*   Restricts click version on py2.6
+*   ash-linux-formula
+    -   Pins docker images to digest for updating by dependabot
```
Comment (lorengordon) [inline CHANGELOG.md]: This is only relevant to the tests, there's no functional changes for the user. We don't need a changelog entry for this type of change.

Comment (eemperor) [reply CHANGELOG.md]: Okay.  I will remove it.

**Comment by eemperor** [comment]
go codebuild go

**Comment by lorengordon** [comment]
@eemperor Good to go, proceed with the release!

**Pull request merged**